client/a8c-for-agencies/components/a4a-migration-offer-v2/style.scss

```diff
@@ -0,0 +1,90 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.a4a-migration-offer-v2 {
+	display: flex;
+	flex-direction: row;
+	gap: 16px;
+
+	border-radius: 4px;
+	border: 1px solid var(--color-primary-5);
+	background-color: var(--color-primary-0);
```

Comment:
This is slightly different from the design as we do not have an official variable for `#F4FBFF`.